src/otg/assets/schemas/study_index.json

```diff
@@ -193,6 +193,26 @@
       "nullable": true,
       "metadata": {}
     },
+    {
+      "name": "qualityControls",
+      "type": {
+        "type": "array",
+        "elementType": "string",
+        "containsNull": true
+      },
+      "nullable": true,
+      "metadata": {}
+    },
+    {
+      "name": "analysisFlags",
```

Comment:
This name can change depending of https://github.com/opentargets/curation/pull/17#pullrequestreview-1794641661